java/src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1411,8 +1460,105 @@ void possibleClusterMember() {
                 utils.testValid("possibleClusterMember with and without optional URI argument");
             }
         }
-    }
 
+        /**
+         * Create ClaimComponent objects
+         */
+        @Nested
+        class ClaimComponentTest {
+            ClaimComponent validComponent = new ClaimComponent()
+                .setName("Hugo Chávez")
+                .setIdentity("Q8440")
+                .addType("Q82955"); // Politician
+
+            @Test
+            void validMininmal() {
+                validComponent.addToModel(model, "https://www.wikidata.org/wiki/Q8440", system);
+                utils.testValid("Create minimal valid ClaimComponent");
+            }
+
+            @Test
+            void validFull() {
```

Comment:
Maybe add optionals to valid claims
    aida:claimId "Claim1" ;
    aida:queryId "Query1" ;
    aida:importance "1.0"^^xsd:double ;
    aida:claimerAffiliation ex:ClaimerAffiliation1, ex:ClaimerAffiliation2 ;
    aida:claimDateTime ex:Time ;
    aida:claimLocation ex:ClaimLocation ;
    aida:identicalClaims ex:identicalClaim1, ex:identicalClaim2 ;
    aida:relatedClaims ex:relatedClaim1, ex:relatedClaim2 ;
    aida:supportingClaims ex:supportingClaim1, ex:supportingClaim2 ;
    aida:refutingClaims ex:refutingClaim1, ex:refutingClaim2 .